src/Dropdown/TagDropdownTrigger.js

```diff
@@ -1,48 +1,97 @@
 import React from "react";
+import PropTypes from "prop-types";
 import AbstractTag from "../Tags/AbstractTag";
-import { DropdownTag } from "../Tags";
+import { ICON_TYPES } from "../constants";
+import Icon from "../Icon";
 
-export default class TagDropdownTrigger extends DropdownTag {
-  constructor(props) {
-    super(props);
-
-    this.ref = React.createRef();
-  }
-
-  render() {
-    const {
-      text,
-      icon,
-      color,
-      skin,
-      isOutline,
-      disabled,
-      ariaDisabled,
-      onInteraction,
-      tabIndex,
-      setRef,
-    } = this.props;
+export default function TagDropdownTrigger({
+  text,
+  icon,
+  color,
+  skin,
+  isOutline,
+  disabled,
+  ariaDisabled,
+  onInteraction,
+  tabIndex,
+  setRef,
+}) {
+  // eslint-disable-next-line react/display-name
+  const handleIcon = () =>
+    icon ? (
+      <Icon
+        type={icon}
+        color="black-75"
+        size="petit"
+        className="lab-tag--left-icon"
+      />
+    ) : undefined;
 
-    return (
-      <AbstractTag
-        className={`lab-tag--dropdown${`${
-          icon ? ` lab-tag--has-left-icon` : ""
-        }`}`}
-        text={text}
-        icon={icon}
-        color={color}
-        skin={skin}
-        isOutline={isOutline}
-        disabled={!ariaDisabled && disabled}
-        ariaDisabled={ariaDisabled}
-        renderPrefix={this.icon()}
-        renderSuffix={this.dropdownIcon()}
-        tabIndex={tabIndex}
-        onInteraction={onInteraction}
-        isDropdown
-        setRef={setRef}
-        role="listbox"
+  const dropdownIcon = () => (
+    <span className="lab-tag__dropdown-icon-wrapper">
+      <Icon
+        type="dropdown-closed"
+        color="black-75"
+        size="petit"
+        className="lab-tag--dropdown-icon"
       />
-    );
-  }
+    </span>
+  );
+
+  return (
+    <AbstractTag
+      className={`lab-tag--dropdown${`${
+        icon ? ` lab-tag--has-left-icon` : ""
+      }`}`}
```

Comment:
> erhm, I think this template string is a bit messed up sweat_smile

Yeah, I think they are =]
We could do something like this:
```suggestion
      className={`lab-tag--dropdown${icon ? ` lab-tag--has-left-icon` : ""}`}
```